scripts/security-checker.mjs

```diff
@@ -86,12 +88,17 @@ class SecurityChecker {
           const alert = this.alertDictionary[key];
 
           if (alert.type === ALERT_TYPES.dependabot) {
-              const isAlertOpened = await this.isDependabotAlertOpened(alert.number);
+              const matchAlertInIssue = alert.issue.body.match(new RegExp(`\`${this.context.repo}\` - Link:\\s*(https:.*/(dependabot|code-scanning)/(\\d+))`));
```

Comment:
It just gets an alert number, as it was in the method `createAlertDictionary`. Please move getting an alert number back to the method `createAlertDictionary`